CMakeLists.txt

```diff
@@ -76,15 +75,8 @@ if(LOFAR_STATION_RESPONSE_DIR AND LOFAR_STATION_RESPONSE_LIB)
     tPredict
```

Comment:
When LofarBeam/EveryBeam is not optional anymore, please also move the names of these tests near the spot where 
LOFAR_DEPENDENT_INTEGRATION_TESTS is used.